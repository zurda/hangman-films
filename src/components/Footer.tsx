import React from 'react'
import styled from '@emotion/styled'
import { colours } from '../style/shared'

const Container = styled.footer`
  width: 100%;
  color: ${colours.diamond};
  background-color: ${colours.pictonBlue};
  font-weight: bold;
  font-family: 'Courier New', Courier, monospace;
  padding: 16px 32px;
  text-align: center;
`

const Description = styled.p`
  margin: 0;
  & a {
    color: inherit;
  }
`

const Footer = () => (
  <Container>
    <Description>
      This project is available on{' '}
      <a href="https://github.com/zurda/hangman-react">Github</a>
    </Description>
  </Container>
)

export default Footer
