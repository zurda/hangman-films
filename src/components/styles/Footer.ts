import styled from 'styled-components';

export const Container = styled.footer`
  width: 100%;
  color: #fff;
  background-color: ${(props) => props.theme.colors.dark};
  font-weight: bold;
  font-family: 'Courier New', Courier, monospace;
  padding: 16px 32px;
  text-align: center;
`

export const Description = styled.p`
  margin: 0;
  & a {
    color: inherit;

    &:focus {
      border: 2px solid ${(props) => props.theme.colors.text};
    }
  }
`