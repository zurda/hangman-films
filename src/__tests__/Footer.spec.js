import React from 'react'
import { create } from 'react-test-renderer'
import Footer from '../components/Footer'

describe('Button component', () => {
  test('Matches the snapshot', () => {
    const footer = create(<Footer />)
    expect(footer.toJSON()).toMatchSnapshot()
  })
})
