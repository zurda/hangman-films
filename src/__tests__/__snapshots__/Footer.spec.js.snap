// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button component Matches the snapshot 1`] = `
<footer
  className="css-3ggn8u"
>
  <p
    className="css-1kdc007"
  />
</footer>
`;
